server/lib/csv.js

```diff
@@ -39,3 +37,19 @@ export const getCollectiveTransactionsCsv = async (collective, { startDate, endD
       logger.error(`Error fetching CSV: ${error.message}`);
     });
 };
+
+export const getCollectiveTransactionsCsv = async (collective, { startDate, endDate } = {}) => {
+  if (parseToBoolean(config.restService.fetchCollectiveTransactionsCsv) === false) {
```

Comment:
Maybe simplify a bit... 🤔 

```suggestion
  if (!parseToBoolean(config.restService.fetchCollectiveTransactionsCsv)) {
```

Reply:
If you're touching this, I would suggest to always test for the condition you want to happen and not the opposite condition:
```ts
if (parseToBoolean(config.restService.fetchCollectiveTransactionsCsv)) {
  return getTransactionsCsv('transactions', collective, { startDate, endDate });
}
```

Reply:
I like the opposite. At the beginning, try to find all the reasons to not do it, return early or throw exceptions. Then, when you're good, proceed without any condition. No nested code and generally easier to read.

Reply:
In different cases, generally a good idea to favor positive assertions versus negative one.